auto_install/install.sh

```diff
@@ -296,6 +311,36 @@ distroCheck(){
 				;;
 			esac
 		;;
+		Alpine)
+		PKG_MANAGER='apk'
+		UPDATE_PKG_CACHE="${PKG_MANAGER} update"
+		PKG_INSTALL="${PKG_MANAGER} --no-cache add"
+		PKG_COUNT="${PKG_MANAGER} --no-cache upgrade -s | grep -sEe ' [0-9]+ packages' | sed -E -e 's/.* ([0-9]+) packages/\1/' || true"
+		CHECK_PKG_INSTALLED="${PKG_MANAGER} --no-cache info -e"
+
+		# install grepcidr manually
+		## install dependencies
+		"${PKG_INSTALL}" build-base make curl tar
+
+		## download binaeries
+		curl -fLo master.tar.gz https://github.com/pivpn/grepcidr/archive/master.tar.gz
```

Comment:
Check whether `grepcidr` is already installed, [this way](https://github.com/pivpn/pivpn/blob/f8cb945af15a1ca0cf063475c6e1557c6e8da06c/auto_install/install.sh#L273) for example.